src/ppom-controller.ts

```diff
@@ -466,6 +491,14 @@ export class PPOMController extends BaseControllerV2<
     this.update((draftState) => {
       draftState.chainStatus = chainStatus;
     });
+  }
+
+  /*
+   * The function adds new network to chainStatus list.
+   */
```

Comment:
[nit] this seems like a copy pasta description for this listener handler

Reply:
good catch! https://github.com/MetaMask/ppom-validator/pull/85/commits/33e89305f1c00ada293979fc90f941181c4d3fae